database/migrations/2019_02_27_061232_create_follower_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateFollowerTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('follower', function (Blueprint $table) {
```

Comment:
is the naming of tables from you, or from the command you executed?

Reply:
It was from the command I executed

Reply:
because in laravel if model is named `Activity` it's table will be named in plural word `activities`. The purpose is that laravel can locate easily the model and table connection. Can you change the name of tables, or create model together with a migration file by running this command `php artisan make:controller Activity -m`. It will create a model and migration file for you.